fideslog/sdk/python/client.py

```diff
@@ -60,6 +62,15 @@ def send(self, event: AnalyticsEvent) -> None:
         Record a new event.
         """
 
+        # There is a Python bug in the default Windows event loop for Python 3.8+. This works around
+        # the issue by changing the default event loop for Windows.
+        if (
+            sys.version_info[0] == 3
+            and sys.version_info[1] >= 8
+            and sys.platform.startswith("win")
+        ):
+            set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

Comment:
```suggestion
        # Work around a Python 3.8+ bug by changing
        # the default event loop for Windows.
        if (
            version_info[0] == 3  # Python v3.x.x
            and version_info[1] >= 8  # Python v3.8.x or greater
            and platform.lower().startswith("win")
        ):
            from asyncio import WindowsSelectorEventLoopPolicy  # type:ignore

            set_event_loop_policy(WindowsSelectorEventLoopPolicy())
```

Reply:
this looks good 👍

Reply:
Since `set_event_loop_policy` is only needed if the os is Windows I moved that to the conditional imports also.